README.md

```diff
@@ -48,25 +48,42 @@ Docker
 
 In order to run own copy of the book with gitook within local container:
```

Comment:
While we are at it, let us fix those typos quickly:
s/run own copy/run your own copy/
s/gitook/gitbook/
s/within local container/within a local container/